src/integrationCapture.ts

```diff
@@ -13,32 +45,39 @@ export default class IntegrationCapture {
      * Captures Integration Ids from cookies and query params and stores them in clickIds object
      */
     public capture(): void {
-        this.captureCookies();
-        this.captureQueryParams();
+        const queryParams = this.captureQueryParams() || {};
+        const cookies = this.captureCookies() || {};
+
+        if (queryParams['fbclid'] && cookies['_fbc']) {
+            // Exclude fbclid if _fbc is present
```

Comment:
Please reference the doc where we get this info from.

Reply:
i think it's this https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/fbp-and-fbc#retrieve-from-fbclid-url-query-parameter